shared/js/project.js

```diff
@@ -316,6 +316,8 @@ var projects = module.exports = {
       case 'studio':
         if (appOptions.level.useContractEditor) {
           return 'algebra_game';
+        } else if (appOptions.skinId === 'hoc2015' || appOptions.skinId === 'infinity') {
```

Comment:
This change itself is simple, but why is this the right logic? Is this intended to be a bandaid for HoC or are we pretty sure this disables the Remix button for all levels which don't support it? I understand this is important to fix quickly, but want to understand if there's follow up work to do here.